src/components/Home.tsx

```diff
@@ -19,8 +18,26 @@ const useStyles = makeStyles((theme: Theme) => ({
     padding: '20px 0',
   },
   main: {
-    background:
-      'url(/assets/trees-right.png) no-repeat 100% 100%/auto 248px, url(/assets/trees-left.png) no-repeat 0 100%/auto 175px, url(/assets/water.png) repeat-x 0 100%/auto 142px, url(/assets/mountain.png) no-repeat 0 100%/auto 233px, url(/assets/far-mountain.png) no-repeat 100% 100%/auto 317px, url(/assets/background.png) no-repeat 100% 0/90% 633px, linear-gradient(to bottom right, rgb(255, 255, 255) 0%, rgb(199, 226, 234) 100%) no-repeat 0 0/auto',
+    [theme.breakpoints.down('xl')]: {
+      background:
+        'url(/assets/trees-right.png) no-repeat 879px 100%/auto 248px, ' +
+        'url(/assets/trees-left.png) no-repeat 0 100%/auto 175px, ' +
+        'url(/assets/water.png) repeat-x 0 100%/auto 142px, ' +
+        'url(/assets/mountain.png) no-repeat 0 100%/auto 233px, ' +
+        'url(/assets/far-mountain.png) no-repeat 1012px 100%/auto 317px, ' +
+        'url(/assets/background.png) no-repeat 100% 0/90% 633px, ' +
+        'linear-gradient(180deg, #FBF9F9 0%, #EFF5EF 100%) no-repeat 0 0/auto',
+    },
+    [theme.breakpoints.up('xl')]: {
+      background:
+        'url(/assets/trees-right.png) no-repeat 100% 100%/auto 248px, ' +
+        'url(/assets/trees-left.png) no-repeat 0 100%/auto 175px, ' +
+        'url(/assets/water.png) repeat-x 0 100%/auto 142px, ' +
+        'url(/assets/mountain.png) no-repeat 0 100%/auto 233px, ' +
+        'url(/assets/far-mountain.png) no-repeat 100% 100%/auto 317px, ' +
+        'url(/assets/background.png) no-repeat 100% 0/90% 633px, ' +
+        'linear-gradient(180deg, #FBF9F9 0%, #EFF5EF 100%) no-repeat 0 0/auto',
+    },
```

Comment:
I typically try to avoid using js string concatenation, but I think the readability is dramatically improved in this case 💯